flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/MesosTaskManagerParameters.java

```diff
@@ -429,26 +418,15 @@ private static double getCpuCores(final Configuration configuration) {
 	}
 
 	private static MemorySize getTotalProcessMemory(final Configuration configuration) {
-		MemorySize legacyTotalProcessMemory = MemorySize.parse(configuration.getInteger(MESOS_RM_TASKS_MEMORY_MB) + "m");
 		MemorySize unifiedTotalProcessMemory = configuration.get(TaskManagerOptions.TOTAL_PROCESS_MEMORY);
 
-		if (configuration.contains(MESOS_RM_TASKS_MEMORY_MB) &&
-			configuration.contains(TaskManagerOptions.TOTAL_PROCESS_MEMORY) &&
-			!legacyTotalProcessMemory.equals(unifiedTotalProcessMemory)) {
+		if (!configuration.contains(TaskManagerOptions.TOTAL_PROCESS_MEMORY)) {
```

Comment:
I'm not sure that the total process memory should be explicitly set here. According to the [FLIP-49 Doc](https://cwiki.apache.org/confluence/display/FLINK/FLIP-49%3A+Unified+Memory+Configuration+for+TaskExecutors). User need either of these three options configured:
- Task Heap Memory and Managed Memory
- Total Flink Memory
- Total Process Memory

After this PR, I think we could directly construct the TaskExecutorResourceSpec from the flinkConfig. @azagrebin Please correct me if I have any misunderstanding.

Reply:
Indeed, we do not need this method `getTotalProcessMemory` anymore.
`TaskExecutorProcessUtils` will parse the options in `createContaineredTaskManagerParameters`.
Subsequently, we do not need `.withTotalProcessMemory(totalProcessMemory)` in `createContaineredTaskManagerParameters`.

Reply:
I will remove .withTotalProcessMemory (totalProcessMemory) in createContaineredTaskManagerParameters and delete the getTotalProcessMemory method

Reply:
@azagrebin Already modified